src/main/java/Database/playlistDsData.java

```diff
@@ -21,12 +22,14 @@ public playlistDsData(Playlist p){
         this.playlist = p;
         library = songLibrary.getInstance();
         users = userList.getInstance();
+        this.privacy = 1;
```

Comment:
I'd suggest adding a comment here to signify what 1 (vs. 0 presumably) signifies wrt privacy (also I thought we were getting rid of playlist privacy controls? is that still the case?

Reply:
Yeah, that is gone from the use cases but I don't want to waste time removing it from the databases until we're totally sure it needs to go.